docs/tutorials/controlling-wedo2-motor.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Controlling a WeDo 2.0 motor
+subject: Networking
+author: "@JorgePe"
+---
+
+* Table of Contents
+{:toc}
+
+## Intro
+
+LEGO Education released the second version of WeDO in the beginning of 2016.
+We'll show how to use the bluez, the linux bluetooth stack, to wireless control a
+WeDo 2.0 motor.
+
+The first WeDO version uses USB so every robot needs to be tethered to a «host»
```

Comment:
Gives it a bit of international flavor. :wink:

Another option would be to use markdown formatting to add emphasis here.

    *italic*
    **bold**

*italic*
**bold**